forms-flow-data-analysis-api/README.md

```diff
@@ -1,64 +1,154 @@
+   * [Step 3 : Running the Application](#running-the-application)
+   * [Step 4 : Verify the Application Status](#verify-the-application-status)
+
+
+## Prerequisites
+
+* For docker based installation [Docker](https://docker.com) need to be installed.
+* Admin access to [Keycloak](../forms-flow-idm/keycloak) server and ensure audience(camunda-rest-api) is setup in Keycloak-bpm server.
+
+## Solution Setup
+
+### Installation
+
+If you are interested in contributing to the project, you can install through docker or locally.
+
+It's recommended to download dev-packages to follow Python coding standards for project like PEP8 if you are interested in contributing to project.
+You installing dev-packages using pip as follows:
+
+```python3 -m pip install -r requirements/dev.txt```
+
+### Keycloak Setup
+
+No specific client creation is required. Audience has been added for clients
+**forms-flow-web** and **forms-flow-bpm**.  
+
+### Environment Configuration
+
+* Make sure you have a Docker machine up and running.
+* Make sure your current working directory is "forms-flow-ai/forms-flow-data-analysis-api".
+* Rename the file [sample.env](./sample.env) to **.env**.
+* Modify the environment variables in the newly created **.env** file if needed. Environment variables are given in the table below,
+* **NOTE : {your-ip-address} given inside the .env file should be changed to your host system IP address. Please take special care to identify the correct IP address if your system has multiple network cards**
+
+> :information_source: Variables with trailing :triangular_flag_on_post: in below table should be updated in the .env file
+
+Variable name | Meaning | Possible values | Default value |
+--- | --- | --- | ---
+`KEYCLOAK_URL`:triangular_flag_on_post:| URL to your Keycloak server || `http://{your-ip-address}:8080`
+`KEYCLOAK_URL_REALM`|The Keycloak realm to use|eg. forms-flow-ai | `forms-flow-ai`
+`KEYCLOAK_WEB_CLIENT_ID`|Client ID for Web to register with Keycloak|eg. forms-flow-web|`forms-flow-web`
+`KEYCLOAK_WEB_CLIENT_ID`|Client ID for Web to register with Keycloak|eg. forms-flow-web|`forms-flow-web`
+`DATA_ANALYSIS_DB_USER` |formsflow data analysis database postgres user|Used on installation to create the database.Choose your own|`general`
+`DATA_ANALYSIS_DB_PASSWORD` |formsflow data analysis database postgres password|Used on installation to create the database.Choose your own|`changeme`
+`DATA_ANALYSIS_DB_NAME` |formsflow data analysis database name|Used on installation to create the database.Choose your own|`dataanalysis`
+`DATA_ANALYSIS_DB_URL` |JDBC DB Connection URL for formsflow||`postgresql://general:changeme@forms-flow-data-analysis-db:5432/dataanalysis`
+`DATA_ANALYSIS_API_BASE_URL`:triangular_flag_on_post:|formsflow.ai Data analysis API URL||`http://{your-ip-address}:5001`
+`MODEL_ID`:triangular_flag_on_post:|formsflow.ai Hugging face model id||`xaqren/sentiment_analysis`
+
```

Comment:
one more env is there
DATABASE_SUPPORT=DISABLED
possible values are DISABLED / ENABLED
forms-flow-data-analysis-api\src\api\config.py
please add that in yml file too